content/en/docs/ambient/install/multicluster/verify/index.md

```diff
@@ -0,0 +1,221 @@
+helloworld-v1-86f77cd7bd-cpxhv  1/1       Running   0          40s
+{{< /text >}}
+
+Wait until the status of `helloworld-v1` is `Running`.
+
+Now, mark the helloworld service in `cluster1` as global so that it can be accessed from other clusters in the mesh:
+
+{{< text bash >}}
+$ kubectl label --context="${CTX_CLUSTER1}" svc helloworld -n sample \
+    istio.io/global="true"
+{{< /text >}}
+
+## Deploy `HelloWorld` `V2`
+
+Deploy the `helloworld-v2` application to `cluster2`:
+
+{{< text bash >}}
+$ kubectl apply --context="${CTX_CLUSTER2}" \
+    -f @samples/helloworld/helloworld.yaml@ \
+    -l version=v2 -n sample
+{{< /text >}}
+
+Confirm the status the `helloworld-v2` pod status:
+
+{{< text bash >}}
+$ kubectl get pod --context="${CTX_CLUSTER2}" -n sample -l app=helloworld
+NAME                            READY     STATUS    RESTARTS   AGE
+helloworld-v2-758dd55874-6x4t8  1/1       Running   0          40s
+{{< /text >}}
+
+Wait until the status of `helloworld-v2` is `Running`.
+
+Now, mark the helloworld service in `cluster2` as global so that it can be accessed from other clusters in the mesh:
+
+{{< text bash >}}
+$ kubectl label --context="${CTX_CLUSTER2}" svc helloworld -n sample \
+    istio.io/global="true"
+{{< /text >}}
+
+## Deploy `curl`
+
+Deploy the `curl` application to both clusters:
+
+{{< text bash >}}
+$ kubectl apply --context="${CTX_CLUSTER1}" \
+    -f @samples/curl/curl.yaml@ -n sample
+$ kubectl apply --context="${CTX_CLUSTER2}" \
+    -f @samples/curl/curl.yaml@ -n sample
+{{< /text >}}
+
+Confirm the status `curl` pod on `cluster1`:
+
+{{< text bash >}}
+$ kubectl get pod --context="${CTX_CLUSTER1}" -n sample -l app=curl
+NAME                             READY   STATUS    RESTARTS   AGE
+curl-754684654f-n6bzf            1/1     Running   0          5s
+{{< /text >}}
+
+Wait until the status of the `curl` pod is `Running`.
+
+Confirm the status of the `curl` pod on `cluster2`:
+
+{{< text bash >}}
+$ kubectl get pod --context="${CTX_CLUSTER2}" -n sample -l app=curl
+NAME                             READY   STATUS    RESTARTS   AGE
+curl-754684654f-dzl9j            1/1     Running   0          5s
+{{< /text >}}
+
+Wait until the status of the `curl` pod is `Running`.
+
+## Verifying Cross-Cluster Traffic
+
+To verify that cross-cluster load balancing works as expected, call the
+`HelloWorld` service several times using the `curl` pod. To ensure load
+balancing is working properly, call the `HelloWorld` service from all
+clusters in your deployment.
+
+Send one request from the `curl` pod on `cluster1` to the `HelloWorld` service:
+
+{{< text bash >}}
+$ kubectl exec --context="${CTX_CLUSTER1}" -n sample -c curl \
+    "$(kubectl get pod --context="${CTX_CLUSTER1}" -n sample -l \
+    app=curl -o jsonpath='{.items[0].metadata.name}')" \
+    -- curl -sS helloworld.sample:5000/hello
+{{< /text >}}
+
+Repeat this request several times and verify that the `HelloWorld` version
+should toggle between `v1` and `v2`:
```

Comment:
I don't think toggle is the right word here. More like both endpoints should be hit.